crates/core/tests/command_input.rs

```diff
@@ -76,7 +77,7 @@ fn run_empty() -> Result<()> {
     let command: CommandInput = "".parse()?;
     dbg!(&command);
     assert!(!command.is_set());
-    command.run("test", "empty")?;
+    command.run("test", "empty", &HashMap::<String, String>::new())?;
```

Comment:
As `Option` also implement `IntoIterator`, I would find using `None` for an empty iterator pretty neat here...

Reply:
This has been done in the last commit